app/src/main/java/org/opencord/aaa/impl/AaaManager.java

```diff
@@ -264,6 +280,29 @@ private void configurePacketCustomizer() {
                 break;
         }
     }
+    private boolean isValidValidator(RADIUS radiusPacket) {
```

Comment:
Correct the formatting. Use tab instead of spaces. Select the added code lines in Eclipse and press Ctrl+F. I see issues in formatting across code files; self review and fix